src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/MsQuicApi.cs

```diff
@@ -89,8 +91,23 @@ static MsQuicApi()
 
         if (OperatingSystem.IsWindows())
         {
-            // Windows ships msquic in the assembly directory.
-            loaded = NativeLibrary.TryLoad(Interop.Libraries.MsQuic, typeof(MsQuicApi).Assembly, DllImportSearchPath.AssemblyDirectory, out msQuicHandle);
+#pragma warning disable IL3000 // Avoid accessing Assembly file path when publishing as a single file
```

Comment:
Why adding this pragma? The UnconditionalSuppress should already suppress this. Feels like this will just make it more likely to get #109958 resurface, whenever someone just mindlessly copies/pastes this code to somewhere else.